src/components/pages/home/jumbotron/index.tsx

```diff
@@ -28,7 +28,7 @@ const Jumbotron: FunctionComponent<JumbotronProps> = ({
         },
       }}
     >
-      <div className="absolute top-0 left-0 bg-gradient-to-r from-purple-500 to-indigo-600 w-full h-2 z-20" />
+      {/* <div className="absolute top-0 left-0 bg-gradient-to-r from-purple-500 to-indigo-600 w-full h-2 z-20" /> */}
```

Comment:
Removed the gradient bar at the top since it didn't look as good with the background image.

Reply:
Having commented code in production seems bad, this should probably be a storybook component variant.

The [jumbotron story](https://egghead-next-storybook-git-lm-lsbanner-backro-53a34a-eggheadio1.vercel.app/?path=/story/components-home-jumbotron--default-story) needs to be updated.